lib/features/digital_guide_view/tabs/adapted_toilets/presentation/adapted_toilets_expansion_tile_content.dart

```diff
@@ -0,0 +1,99 @@
+  const AdaptedToiletsExpansionTileContent({
+    required this.digitalGuideResponseExtended,
+  });
+
+  final DigitalGuideResponseExtended digitalGuideResponseExtended;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncAdaptedToilets = ref
+        .watch(GetAdaptedToiltesProvider(digitalGuideResponseExtended.levels));
+    return asyncAdaptedToilets.when(
+      data: (adaptedToiletsData) => _AdaptedToiletsExpansionTileContent(
+        levels: digitalGuideResponseExtended.levels
+            .where(
+              (level) =>
+                  adaptedToiletsData.containsKey(level.id) &&
+                  adaptedToiletsData[level.id]!.isNotEmpty,
+            )
+            .toList(),
+        adaptedToiletsData: adaptedToiletsData,
+      ),
+      error: (error, stackTrace) => MyErrorWidget(error),
+      loading: () => const Center(
+        child: CircularProgressIndicator(),
+      ),
+    );
+  }
+}
+
+class _AdaptedToiletsExpansionTileContent extends ConsumerWidget {
+  const _AdaptedToiletsExpansionTileContent({
+    required this.levels,
+    required this.adaptedToiletsData,
+  });
+
+  final List<Level> levels;
+  final Map<int, List<AdaptedToilet>> adaptedToiletsData;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        top: DigitalGuideConfig.heightMedium,
+        left: DigitalGuideConfig.heightMedium,
+        right: DigitalGuideConfig.heightMedium,
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: levels.map((level) {
+          return Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                level.translations.plTranslation.name,
+                style: context.textTheme.title,
+              ),
+              const SizedBox(
+                height: DigitalGuideConfig.heightSmall,
+              ),
+              ListView.separated(
+                physics: const NeverScrollableScrollPhysics(),
+                itemBuilder: (context, index) => DigitalGuideNavLink(
+                  onTap: () async {
+                    await ref.navigateAdaptedToiletDetails(
+                      adaptedToiletsData[level.id]![index],
+                    );
+                  },
+                  text: adaptedToiletsData[level.id]![index]
+                      .getDescription(context),
+                ),
+                separatorBuilder: (context, index) => const SizedBox(
+                  height: DigitalGuideConfig.heightMedium,
+                ),
+                itemCount: adaptedToiletsData[level.id]!.length,
+                shrinkWrap: true,
+              ),
+              const SizedBox(
+                height: DigitalGuideConfig.heightMedium,
+              ),
+            ],
+          );
+        }).toList(),
```

Comment:
Ilist?

Reply:
I don't think it's possible here unfortunately because Column Widget expects mutable List<Widget> instead of IList<Widget>. It's the same case with ListView.